File: 2nd sem/assignment 8/p2.c
#include<stdio.h>
/*don't give any input in command line.so default argc is 1*/
void main(int argc,int n )
{
	if(argc==1)
	{
		if(printf("Enter a no.: ")){}
		if(scanf("%d",&n)){}
	}
	if(argc<=n)
	{
		if(printf(" %d",argc)){}
		if((main(argc+1,n),1)){}
	}	
}

File: 3rd sem/IT Lab/assignment 5/p2.py
a=int(input("Enter the range: "))

b=0;c=1;d=1
while(c<a):
    b=c
    c=d
    d=b+c
    if(c>a):
        c=a+1
    for i in range(b+1,c):
        print(i)
   

